pontoon-to-webext.js

```diff
@@ -0,0 +1,164 @@
+#! /usr/bin/env node
+
+/* eslint-disable promise/avoid-new */
+
+const propertiesParser = require('properties-parser');
+const path = require('path');
+const FS = require('q-io/fs');
+const argv = require('minimist')(process.argv.slice(2));
+
+const Habitat = require('habitat');
```

Comment:
:+1: I think we should add a little note in the README.md or whatever about creating an .env file and adding the `SUPPORTED_LOCALES` config (or whatever ENV variables we're trying to import/load via **habitat**).